src/main/java/org/seqdoop/hadoop_bam/VCFRecordReader.java

```diff
@@ -84,11 +84,11 @@
 		reader = new AsciiLineReader(ins);
         it = new AsciiLineReaderIterator(reader);
 		
-		final Object h = codec.readHeader(it);
-		if (!(h instanceof FeatureCodecHeader) || !(((FeatureCodecHeader)h).getHeaderValue() instanceof VCFHeader))
+		final FeatureCodecHeader h = codec.readHeader(it);
```

Comment:
I wonder.. any idea what was the original reason for having Object there? Was that required by an old HTSJDK API?